frontend_tests/node_tests/rendered_markdown.js

```diff
@@ -251,7 +251,7 @@ run_test("timestamp without time", () => {
 
 run_test("timestamp", ({mock_template}) => {
     mock_template("markdown_timestamp.hbs", true, (data, html) => {
-        assert.deepEqual(data, {text: "Thu, Jan 1 1970, 12:00 AM"});
+        assert.deepEqual(data, {text: "Thu, Jan 01, 1970, 12:00 AM"});
```

Comment:
Can we avoid the format change to using the leading 0 in `01` here? 

@alya can confirm, but I think we want the cleaner format everywhere, and in particular this one (which is the "global times" feature).

Reply:
Yep, no leading zeros anywhere, please!

Reply:
Got it!

Reply:
@alya, I just wanted to double check.

We **will not** have leading zeros for:
1. day numbers (1 Jan)
2. 12-hour time format (1:30 AM)

We **will have** leading zeros for:
1. 24-hour time format (01:30)

Reply:
Yes, that's correct.